setup.py

```diff
@@ -76,13 +76,15 @@ def run_tests(self):
     classifiers=[
         'Development Status :: 5 - Production/Stable',
         'Framework :: Django',
-        'Framework :: Django :: 1.8',
-        'Framework :: Django :: 1.9',
+        'Framework :: Django :: 1.11',
+        'Framework :: Django :: 2.1',
+        'Framework :: Django :: 2.2',
         'Programming Language :: Python :: 2',
```

Comment:
I think this line can be removed
And  line 34 too

Reply:
Do you mean lines `'Programming Language :: Python :: 2',` and `'Programming Language :: Python :: 3',`?

Reply:
Nvm. Ignore me. Stupid comment before I got my coffee this morning